app/graphql/types/api/mutation_type.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+module Types
+  module Api
+    class MutationType < Types::BaseObject
+      field :login_user, mutation: Mutations::LoginUser
+      field :register_user, mutation: Mutations::RegisterUser
+
+      field :update_organization, mutation: Mutations::Organizations::Update
+
+      field :create_billable_metric, mutation: Mutations::BillableMetrics::Create
+      field :destroy_billable_metric, mutation: Mutations::BillableMetrics::Destroy
+      field :update_billable_metric, mutation: Mutations::BillableMetrics::Update
+
+      field :create_adjusted_fee, mutation: Mutations::AdjustedFees::Create
+      field :destroy_adjusted_fee, mutation: Mutations::AdjustedFees::Destroy
+
+      field :create_plan, mutation: Mutations::Plans::Create
+      field :destroy_plan, mutation: Mutations::Plans::Destroy
+      field :update_plan, mutation: Mutations::Plans::Update
+
+      field :create_customer, mutation: Mutations::Customers::Create
+      field :destroy_customer, mutation: Mutations::Customers::Destroy
+      field :update_customer, mutation: Mutations::Customers::Update
+      field :update_customer_invoice_grace_period, mutation: Mutations::Customers::UpdateInvoiceGracePeriod
+
+      field :generate_customer_portal_url, mutation: Mutations::CustomerPortal::GenerateUrl
```

Comment:
Could we consider move this mutation out of the `CustomerPortal` namespace?

What do you think about `Mutations::GenerateCustomerPortalUrl`?